### DISCUSSION TITLE
【beginner】I found a place to fix it in the doc, but I don't know where the file is in the folder.

### DISCUSSION THREAD

**Discussion by anoonoll** [vercel/next.js#32809] [Help] [answered]
I'm a newbie trying to send a pull request to OSS for the first time.

I want to send a pull request for the ↓ page.
https://nextjs.org/learn/basics/dynamic-routes/render-markdown

Where can I find the page to modify this page in next.js?

I fixed it with git and finished until clone, but then I can't find the file to fix.

I would appreciate it if you could answer.

**Comment by alex-semenyuk** [comment, marked as answer]
I believe the site build is not under github. That's why you can't find it

**Reply by anoonoll** [reply]
I see!
Thank you so much XD
